File: boj/swift/1008.swift
//
//  1008번: A/B
//  https://www.acmicpc.net/problem/1008
//
//  Created by WhiteHyun on 2021/11/20.
//

import Foundation

let inputArray = readLine()!.components(separatedBy: " ").map { Double($0)! }
print(inputArray[0] / inputArray[1])

File: boj/bronze4/1008.py
# 1008번: A/B
# https://www.acmicpc.net/problem/1008
# Version: Python 3.8.12


from sys import stdin

read = stdin.readline

if __name__ == "__main__":
    a, b = map(int, read().split())
    print(a / b)
